django/db/models/query.py

```diff
@@ -1701,6 +1701,9 @@ def order_by(self, *field_names):
         obj.query.add_ordering(*field_names)
         return obj
 
+    def _in_place_empty_order_by(self):
+        self.query.clear_ordering(force=True, clear_default=True)
+
```

Comment:
I don't think this warrants a `QuerySet` level method. Calling `clear_ordering` directly should do. We also want to avoid force clearing if possible as users might compose a complex query through `Prefetch(queryset)` where distinct and / or slicing require `ORDER BY` to remain present to preserve the semantic of the query.

Reply:
Ok that was one the options I suggested. Will do.